frontend/ui/data/css_tweaks.lua

```diff
@@ -1026,11 +1044,54 @@ This tweak can be duplicated as a user style tweak when books contain footnotes
 {
     -cr-hint: footnote-inpage;
     margin: 0 !important;
-    font-size: 0.8rem !important;
+}
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: inpage-footnote;
+        font-size: 0.8rem !important;
+}
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: extended-inpage-footnote;
+        font-size: 0.8rem !important;
 }
             ]],
             separator = true,
         },
+        {
+            title = _("In-page footnote extension"),
+            {
+                id = "following-footnote-inpage_all",
```

Comment:
(Dunno if `_all` is standardized elsewhere, but `_any` -or nothing- feels more accurate?)

Reply:
changed to `_any` (and adjusted the id prefix to `extend-footnote-inpage_following-inpage-footnote_` which I think fits with the existing naming scheme based on the `-cr-hint` / `-cr-only-if` that is applied or used.

Reply:
Feels a bit long though :) (related to all the other ids).
`extend-footnote-inpage_following-inpage-footnote_ `
May be just ("following inpage footnote" seems implied with "extend footnote inpage"):
`extend-footnote-inpage_any`
`extend-footnote-inpage_until_next_heading`
?
(Not enough energy to review the rest this evening.)

Reply:
Looks ok, but still waiting for these to get shorter :)

Reply:
sorry, had forgotten about this. They are shorter now :+1: